gslib/commands/cp.py

```diff
@@ -451,176 +445,166 @@
-  - when decompressing data being downloaded (when the data has
-    Content-Encoding:gzip, e.g., as happens when uploaded using gsutil cp -z
-    or gsutil cp -Z)
-  - when running integration tests (using the gsutil test command)
+  - when compressing data to be uploaded (see the ``-z`` and ``-Z`` options)
+  - when decompressing data being downloaded (for example, when the data has
+    ``Content-Encoding:gzip`` as a result of being uploaded
+    using gsutil cp -z or gsutil cp -Z)
+  - when running integration tests using the gsutil test command
 
-  In these cases it's possible the temp file location on your system that
+  In these cases, it's possible the temporary file location on your system that
   gsutil selects by default may not have enough space. If gsutil runs out of
-  space during one of these operations (e.g., raising
+  space during one of these operations (for example, raising
   "CommandException: Inadequate temp space available to compress <your file>"
-  during a gsutil cp -z operation), you can change where it writes these
-  temp files by setting the TMPDIR environment variable. On Linux and macOS
-  you can do this either by running gsutil this way:
+  during a ``gsutil cp -z`` operation), you can change where it writes these
+  temp files by setting the TMPDIR environment variable. On Linux and macOS,
+  you can set the variable as follows:
 
     TMPDIR=/some/directory gsutil cp ...
 
-  or by adding this line to your ~/.bashrc file and then restarting the shell
+  You can also add this line to your ~/.bashrc file and restart the shell
   before running gsutil:
 
     export TMPDIR=/some/directory
 
-  On Windows 7 you can change the TMPDIR environment variable from Start ->
+  On Windows 7, you can change the TMPDIR environment variable from Start ->
   Computer -> System -> Advanced System Settings -> Environment Variables.
-  You need to reboot after making this change for it to take effect. (Rebooting
-  is not necessary after running the export command on Linux and macOS.)
+  You need to reboot after making this change for it to take effect. Rebooting
+  is not necessary after running the export command on Linux and macOS.
 """
 
 _COPYING_SPECIAL_FILES_TEXT = """
-<B>SYNCHRONIZING OVER OS-SPECIFIC FILE TYPES (SYMLINKS, DEVICES, ETC.)</B>
+<B>SYNCHRONIZING OVER OS-SPECIFIC FILE TYPES (SUCH AS SYMLINKS AND DEVICES)</B>
 
   Please see the section about OS-specific file types in "gsutil help rsync".
-  While that section was written specifically about the rsync command, analogous
-  points apply to the cp command.
+  While that section refers to the ``rsync`` command, analogous
+  points apply to the ``cp`` command.
 """
 
 _OPTIONS_TEXT = """
 <B>OPTIONS</B>
-  -a canned_acl  Sets named canned_acl when uploaded objects created. See
-                 "gsutil help acls" for further details.
+  -a canned_acl  Sets the specific ``canned_acl`` when the uploaded objects are
```

Comment:
Suggestion: "Applies the specific ``canned_acl`` to uploaded objects"

Reply:
Done